src/app/services/recovery.service.ts

```diff
@@ -379,65 +385,55 @@ export class RecoveryService {
   }
 
   private getAddressData(address: any, coin: string, network: string, cb: Function): any {
```

Comment:
please change `getAddressData` and `getAddressInfo` for better names.

Reply:
`checkAddress()` and `getAddressTxos()` are the new names